pkg/registry/apis/folders/conversions.go

```diff
@@ -114,16 +114,6 @@ func UnstructuredToLegacyFolder(item unstructured.Unstructured, orgID int64) (*f
 		// meta.GetUpdatedTimestamp() but it currently gets overwritten in prepareObjectForStorage().
 		Updated: createdTime,
 		OrgID:   orgID,
-
-		// This will need to be restructured so the full path is looked up when saving
-		// it can't be saved in the resource metadata because then everything must cascade
-		// nolint:staticcheck
-		Fullpath: meta.GetFullPath(),
```

Comment:
IIUC, these are not needed because they are *QUERY* results, not write commands 🤔